server/services/utils/helpers.js

```diff
@@ -26,3 +26,38 @@ export const isIndexNotFoundError = (err) => {
       'Configured indices are not found: [.opendistro-alerting-config]'
   );
 };
+
+export function getClientBasedOnDataSource(
+  context,
+  dataSourceEnabled,
+  dataSourceId,
```

Comment:
since the `dataSourceId` is a field inside the `request` we don't need to ask for it explicitly

Reply:
done